src/DrupalCheck.php

```diff
@@ -33,9 +33,16 @@ public static function getConfigurableOptions(): OptionsResolver
   {
       $resolver = new OptionsResolver();
       $resolver->setDefaults([
-        'drupal_root' => null,
+        'drupal_root' => NULL,
+        'deprecations' => NULL,
+        'analysis' => NULL,
+        'php8' => NULL,
```

Comment:
I think we can have typed default values instead of NULL.

```suggestion
        'drupal_root' => '',
        'deprecations' => true,
        'analysis' => false,
        'php8' => false,
```